ldoc/html/ldoc_new_css.lua

```diff
@@ -0,0 +1,292 @@
+return [[
+body { 
+    color: #47555c;
+    font-size: 16px;
+    font-family: "Open Sans", sans-serif;
+    margin: 0;
+    background: #eff4ff;
+}
+
+a:link { color: #008fee; }
+a:visited { color: #008fee; }
+a:hover { color: #22a7ff; }
+
+h1 { font-size:26px; font-weight: normal; }
+h2 { font-size:22px; font-weight: normal; }
+h3 { font-size:18px; font-weight: normal; }
+h4 { font-size:16px; font-weight: bold; }
+
+hr {
+    height: 1px;
+    background: #c1cce4;
+    border: 0px;
+    margin: 15px 0;
+}
+
+code, tt {
+    font-family: monospace;
+}
+span.parameter {
+    font-family: monospace;
+    font-weight: bold;
+    color: rgb(99, 115, 131);
+}
+span.parameter:after {
+    content:":";
+}
+span.types:before {
+    content:"(";
+}
+span.types:after {
+    content:")";
+}
+.type {
+    font-weight: bold; font-style:italic
+}
+
+p.name {
+    font-family: "Andale Mono", monospace;
+}
+
+#navigation {
+    float: left;
+    background-color: white;
+    border-right: 1px solid #d3dbec;
+    border-bottom: 1px solid #d3dbec;
+
+    width: 14em;
+    vertical-align: top;
+    overflow: visible;
+}
+
+#navigation br {
+    display: none;
+}
+
+#navigation h1 {
+    background-color: white;
+    border-bottom: 1px solid #d3dbec;
+    padding: 15px;
+    margin-top: 0px;
+    margin-bottom: 0px;
+}
+
+#navigation h2 {
+    font-size: 18px;
+    background-color: white;
+    border-bottom: 1px solid #d3dbec;
+    padding-left: 15px;
+    padding-right: 15px;
+    padding-top: 10px;
+    padding-bottom: 10px;
+    margin-top: 30px;
+    margin-bottom: 0px;
+}
+
+#content h1 {
+    background-color: #2c3e67;
+    color: white;
+    padding: 15px;
+    margin: 0px;
+}
+
+#content h2 {
+    background-color: #6c7ea7;
+    color: white;
+    padding: 15px;
+    padding-top: 15px;
+    padding-bottom: 15px;
+    margin-top: 0px;
+}
+
+#content h2 a {
+    background-color: #6c7ea7;
+    color: white;
+    text-decoration: none;
+}
+
+#content h2 a:hover {
+    text-decoration: underline;
+}
+
+#content h3 {
+    font-style: italic;
+    padding-top: 15px;
+    padding-bottom: 4px;
+    margin-right: 15px;
+    margin-left: 15px;
+    margin-bottom: 5px;
+    border-bottom: solid 1px #bcd;
+}
+
+#content h4 {
+    margin-right: 15px;
+    margin-left: 15px;
+    border-bottom: solid 1px #bcd;
+}
+
+#content pre {
+    margin: 15px;
+}
+
+pre {
+    background-color: rgb(50, 55, 68);
+    color: white;
+    border-radius: 3px;
+    /* border: 1px solid #C0C0C0; /* silver */
+    padding: 15px;
+    overflow: auto;
+    font-family: "Andale Mono", monospace;
+}
+
+#content ul pre.example {
+    margin-left: 0px;
+}
+
+table.index {
+/* border: 1px #00007f; */
+}
+table.index td { text-align: left; vertical-align: top; }
+
+#navigation ul
+{
+    font-size:1em;
+    list-style-type: none;
+    margin: 1px 1px 10px 1px;
+}
+
+#navigation li {
+    text-indent: -1em;
+    display: block;
+    margin: 3px 0px 0px 22px;
```

Comment:
I think it is better without the left margin of 22px, but that might depend on how long your names are.

Reply:
I agree with this one. Would you consider submitting this tweak and anything else you found useful to accommodate various content from the AwesomeWM theme or elsewhere as a follow-up PR?